frappe/public/js/frappe/form/controls/control.js

```diff
@@ -41,6 +41,8 @@ import './duration';
 import './icon';
 
 frappe.ui.form.make_control = function (opts) {
+	if (opts.df.fieldtype == "Long Int")
+		return new frappe.ui.form["ControlInt"](opts);
```

Comment:
Or we can do something like [this](https://github.com/frappe/frappe/blob/a096b85b52b84a26e9610c4c561f7d708eb2dd86/frappe/public/js/frappe/form/controls/data.js#L266) to make `ControlLongInt` searchable.